benchmarks/lib/src/jmh/java/com/amazon/corretto/crypto/provider/benchmarks/AesCfbBase.java

```diff
@@ -0,0 +1,36 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.corretto.crypto.provider.benchmarks;
+
+import java.security.Key;
+import javax.crypto.Cipher;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+public class AesCfbBase {
```

Comment:
Some context here -- this and its deriving classes are basically identical to [the AES GCM benchmarks](https://github.com/corretto/amazon-corretto-crypto-provider/blob/main/benchmarks/lib/src/jmh/java/com/amazon/corretto/crypto/provider/benchmarks/AesGcmBase.java). I didn't want to overly crowd this PR by touching unrelated files, but I plan on following this up with another PR consolidating AES {GCM, CBC, CFB} into a unified set of base/derivative classes once this PR has been merged.

Reply:
https://github.com/corretto/amazon-corretto-crypto-provider/pull/460